pylibs/stress_tests/BaseStressTest.py

```diff
@@ -116,6 +116,9 @@ def report(self):
             if stress_result_fd is not sys.stdout:
                 stress_result_fd.close()
 
+        if self.result.failed:
+            raise RuntimeError("Stress test failed: \n%s" % "\n".join("\t" + msg for msg in self.result._fail_msgs))
```

Comment:
This seems to be a string concatenation. May consider using `+` instead of a formatted string?

Reply:
```suggestion
            raise RuntimeError("Stress test failed: \n" + "\n".join("\t" + msg for msg in self.result._fail_msgs))
```